deploy_piezo.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: piezo-web-app
+  labels:
+    run: piezo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: piezo
+  template: 
+    metadata:
+      labels:
+        run: piezo
+    spec:
+      containers:
+      - name: piezo
+        image: host-172-16-113-64.nubes.stfc.ac.uk/piezo-resources/web_app:latest
```

Comment:
This (and other) references to `host-172-16-113-64.nubes.stfc.ac.uk` needs to be pulled out into a `var` or similar